### PR TITLE
RAS-1305 Introduce non root application user

### PR DESCRIPTION
**Pull request by arroyoAle**
# What and why?
Adds an app specific user so we don't have it run on the root user - no changes to the image as the one we are currently using has the least number of vulnerabilities
# How to test?
- deploy this PR to your env
- run tests and make sure frontstage still works
- `kubectl exec -it <pod-name>` into the pod and run `ps aux` to make sure that the new `appuser` is running the app instead of the root user
# Jira
[RAS-1305](https://jira.ons.gov.uk/browse/RAS-1305)

### CODE REVIEW

File: _infra/docker/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,16 @@
 # Dockerfile specifically for kubernetes
 FROM python:3.11-slim
 
-RUN apt update && apt install -y build-essential curl gunicorn
+RUN apt update && apt install -y build-essential curl gunicorn procps
```
Comment (arroyoAle) [inline _infra/docker/Dockerfile:4]: This is the only way I've found to install `ps` now that we are the new user - it will be removed after testing

**Review by anwilkie — APPROVED** [review]
Ran the command and confirmed that `appuser` is running the app. Tests passed, too. Approved.

**Review by pricem14pc — APPROVED** [review]
Don't forget to revert the `procps` package install

**Pull request merged**